Docs/reference/content/upgrading.md

```diff
@@ -11,9 +11,6 @@ title = "Upgrading"
 
 ## Breaking Changes
 
-For most users there should be no breaking changes in version 2.8.0 of the driver.
+There should be no breaking changes in version 2.9.0 of the driver.
 
-Because we have updated several external dependencies to newer versions, you might encounter compatibility
-issues if your application depends on different versions of those dependencies. This might apply to you
-if you depend on System.Runtime.InteropServices.RuntimeInformation or DnsClient. Note that even if your
-application does not depend on these directly, it is possible that you depend on them indirectly.
+Starting with the 2.9.0 release, retryable writes are enabled by default.
```

Comment:
Since it's not a breaking change (I think so for now), should some title be added before this line?
Like probably:
`### Minor Changes`?